locksmith/__tests__/controllers/v2/ticketsController.test.ts

```diff
@@ -67,6 +69,29 @@ describe('sign endpoint', () => {
     expect(response.status).toBe(200)
   })
 
+  it('marks ticket as checked-in without auth fails', async () => {
+    expect.assertions(1)
+    const response = await request(app).put(
+      `/v2/api/ticket/${network}/lock/${lockAddress}/key/${tokenId}/check`
+    )
+
+    expect(response.status).toBe(500)
```

Comment:
It should not return 500 but 401

Reply:
ha, we probably need to change this in another PR then.
Don't worry about it here.